packages/graphql/tests/tck/tck-test-files/directives/auth/arguments/where/where.test.ts

```diff
@@ -35,14 +35,14 @@ describe("Cypher Auth Where", () => {
             type User {
                 id: ID
                 name: String
-                posts: [Post] @relationship(type: "HAS_POST", direction: OUT)
-                content: [Search] @relationship(type: "HAS_POST", direction: OUT) # something to test unions
+                posts: [Post!]! @relationship(type: "HAS_POST", direction: OUT)
+                content: [Search!]! @relationship(type: "HAS_POST", direction: OUT) # something to test unions
             }
 
             type Post {
                 id: ID
                 content: String
-                creator: User @relationship(type: "HAS_POST", direction: IN)
+                creator: User! @relationship(type: "HAS_POST", direction: IN)
```

Comment:
I've noticed that you've changed all relationships, even single ones, to be non-nullable. Just to confirm, the following wouldn't throw and error right?

```graphql
creator: User @relationship(type: "HAS_POST", direction: IN)
```

Reply:
I wrote this comment early in my review - I now see that the example I gave would be perfectly valid, I just found it interesting how you've replaced all nullable fields with non-nullable fields in the tests. 🙂

Reply:
True I believe I have replaced if not most nullable fields with non-nullable fields, and this was probably the find and replace regex I used.